README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
-here are some things you will need to know:
-* The default port for the Minecraft server is: `25565`.
-* If you wish to run multiple world servers using this script, you will
-  want to open a range of ports (for example `25565 - 25575`).
+## Installation
+### Downloading the script
+The easiest way to download the script is to make a clone of the [git
+repository](https://github.com/sandain/MinecraftServerControlScript.git). 
+You must have git installed first. To install git:
 
-See the [iptables.rules](iptables.rules)
-file for a very basic set of rules that you can use with the Iptables firewall.
+        sudo apt-get install git
+        
+Then:
+
+        git clone https://github.com/sandain/MinecraftServerControlScript.git
+        
+Note that it will be downloaded into the current directory which you are working
+ in. 
+
+##### Other ways to download
+
+* Get the latest stable [release]
+(https://github.com/sandain/MinecraftServerControlScript/releases).
+
+* Get the development version as a [zip file]
+(https://github.com/sandain/MinecraftServerControlScript/archive/master.zip):
+
+        wget https://github.com/sandain/MinecraftServerControlScript/archive/master.zip
+
+
+
+### Configuration
+
+Navigate to the `MinecraftServerControlScript` directory that you just 
+downloaded. 
+Configuration can be done with the included Makefile in Debian and
+Ubuntu like environments by running:
+
+        sudo make install
+Then, type
+
+        chmod -R u+w /opt/mscs
+        chown -R minecraft:minecraft /opt/mscs
+This will give the user you created in the config 
+(by default, the user `minecraft`) 
+access to write in the `/opt/mscs` folder. 
+If you configured MSCS manually when you installed the script, 
+then replace `minecraft` with the name of the user you made.
+
+That's it!
+If you wish to configure the script manually, please visit the [wiki page]
+(https://github.com/sandain/MinecraftServerControlScript/wiki/Manual-Configuration).
+
+
+## Getting started guide
+So you successfully installed the script--great! 
+
+At first, you probably want to [create a new world](#creating-a-new-world) or 
+[import an existing world](#importing-an-existing-world) into the script. 
+
+Then, you should adjust the [world properties](#adjusting-world-properties), 
+adjust the [global server settings](#adjusting-global-server-settings), and 
+enable any other 
+[server software](#enabling-forge-bungeecord-and-other-server-software) as needed.
+
+### Creating a new world
+The command to create a new world is:
+
+        mscs create [world] [port] <ip>
+  
+Where `world` is the name of the world you specify, 
+and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds 
+across different servers. 
+
+Afterwards, simply start the server via `mscs start [world]` where `world` 
+is the name of the world. 
+
+**Finally, accept the EULA**.
```

Comment:
>Finally, accept the EULA

I think this can just be "Accept the EULA"
Also if I remember correctly, you have to manually start the world again after editing `eula.txt`